### PR TITLE
BUGFIX - error message text and cypress tests

### PR DESCRIPTION
**Pull request by s-barrah**
- Fix submit error message text
- Adds new tests for field errors

### CODE REVIEW

File: cypress/integration/submit_spec.js [hunk 1]
```diff
@@ -50,6 +50,12 @@ describe('e2e test', () => {
         cy.get('#field-input--firstname').clear()
         cy.get('#field-input--lastname').click()
         cy.get('#field-error--firstname>span').should('contain', 'Please fill in your first name')
+        cy.get('#field-input--firstname').type('Test@')
```
Comment (KrupaPammi) [inline cypress/integration/submit_spec.js]: there shouldn't be back slash  \  in the first name error message according to the pattern

File: cypress/integration/submit_spec.js [hunk 2]
```diff
@@ -59,6 +65,12 @@ describe('e2e test', () => {
         cy.get('#field-input--lastname').clear()
         cy.get('#field-input--firstname').click()
         cy.get('#field-error--lastname>span').should('contain', 'Please fill in your last name')
+        cy.get('#field-input--lastname').type('test@')
+        cy.get('#field-error--lastname>span').should('contain', 'This field only accepts alphanumeric characters and , . ( ) / & \' - ')
```
Comment (KrupaPammi) [inline cypress/integration/submit_spec.js]: same here there's double appostrophes and a back slash in the last name error message which is not there in the pattern.

**Comment by s-barrah** [comment]
> Reviewed the cyress tests there's no back slash \ in the first name error message
> There's double apostrophe ' ' and a back slash in the last name error message which the field doesn't allow.

Not the problem. The problem was that I was not clearing the input field before entering a new value.

File: cypress/integration/submit_spec.js (hunk 2 above)
Comment (s-barrah) [reply cypress/integration/submit_spec.js]: Not the problem. The problem was that I was not clearing the input field before entering a new value.

Comment (s-barrah) [reply cypress/integration/submit_spec.js]: The backslash is to escape the `'` which is enclosed in single quotes.

**Comment by KrupaPammi** [comment]
Ah right sorry, I keep forgetting that

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**